docs/features/publishing-binaries-using-maven-publish-plugin.md

```diff
@@ -0,0 +1,83 @@
+### Publishing binaries using maven-publish plugin
+
+See also "[How Shipkit Works](/docs/how-shipkit-works.md)" documentation index.
+Please help us with docs and submit a PR with improvements!
+
+You have the code but your users need compiled and tested binaries so that they can use your code!
+
+#### Intro
+
+Shipkit will automatically publish binaries to [Bintray](https://bintray.com) repository.
+There is also an option to publish your artifacts to your internal Maven repository using [Maven Publish Plugin](https://docs.gradle.org/current/userguide/publishing_maven.html). 
+
+#### Nitty gritty
+
+Applying "[org.shipkit.java](https://plugins.gradle.org/plugin/org.shipkit.java)" Gradle plugin will automatically apply "[org.shipkit.bintray](https://plugins.gradle.org/plugin/org.shipkit.bintray) plugin what is not expected in this case.
```

Comment:
This dobule quote: `"[org.shipkit.bintray]` is not closed anywhere.

Additionally, looking at this diff and nearby lines it seems more consistent to not enclose any clickable links in double quotes.

Reply:
fixed